src/sentry/event_manager.py

```diff
@@ -859,6 +864,7 @@ def save(self, project, raw=False):
                 is_new=is_new,
                 is_sample=is_sample,
                 is_regression=is_regression,
+                is_new_group_environment=is_new_group_environment,
```

Comment:
I'm not sure there is much value in passing through the instance ID here or any other more detailed data.